__tests__/PropTypesDevelopmentStandalone-test.js

```diff
@@ -121,7 +124,8 @@ describe('PropTypesDevelopmentStandalone', () => {
       );
       expect(console.error.calls.argsFor(0)[0]).toEqual(
         'Warning: Failed prop type: testComponent: prop type `foo` is invalid; ' +
-        'it must be a function, usually from the `prop-types` package, but received `undefined`.'
+        'it must be a function, usually from the `prop-types` package, but received `undefined`.' +
+        warning
```

Comment:
please revert the test changes here; DRY should not be applied to tests, you end up in a desert.